xbmc/games/addons/GameClientJoystick.cpp

```diff
@@ -171,6 +171,58 @@ bool CGameClientJoystick::OnAccelerometerMotion(const std::string& feature, floa
   return bHandled;
 }
 
+bool CGameClientJoystick::OnWheelMotion(const std::string& feature, float position, unsigned int motionTimeMs)
+{
+  bool bHandled = false;
+
+  game_input_event event;
+
+  std::string controllerId = m_controller->ID();
+
+  event.type                    = GAME_INPUT_EVENT_AXIS;
+  event.port                    = m_port;
+  event.controller_id           = controllerId.c_str();
+  event.feature_name            = feature.c_str();
+  event.axis.position           = position;
+
+  try
+  {
+    bHandled = m_dllStruct->InputEvent(&event);
+  }
+  catch (...)
+  {
+    CLog::Log(LOGERROR, "GAME: %s: exception caught in InputEvent()", m_gameClient->ID().c_str());
```

Comment:
Would it make sense to log that the exception happend in `OnWheelMotion()` to be able to distinguish it from exceptions caught in `OnThrottleMotion()`?

Reply:
Input logging is already way verbose, but adding the function name can't hurt.